### ISSUE TITLE
import: support --force

### ISSUE THREAD

**Issue by LahiLuk** [iterative/dvc#9629]
# Bug Report

## Description

Running `dvc import url path` when the file/s already exist locally gives direction to re-run with `--force` to override. However, it seems that `--force` is not a valid argument to `dvc import`.

### Reproduce

1. run `dvc import url path` or `dvc get url path` --> downloads the file/s locally
2. run `dvc import url path` --> results in `
ERROR: failed to import path from url. - The file file_name already exists locally. 
To override it, re-run with '--force'.
`
3. run `dvc import --force url path` --> results in `ERROR: unrecognized arguments: --force`

### Expected

`--force` should be recognized as an attribute to `import`.

### Environment information

```
DVC version: 3.0.0 (pip)
------------------------
Platform: Python 3.10.6 on Linux-5.19.0-45-generic-x86_64-with-glibc2.35
Subprojects:
	dvc_data = 1.11.0
	dvc_objects = 0.23.0
	dvc_render = 0.5.3
	dvc_task = 0.3.0
	scmrepo = 1.0.3
Supports:
	http (aiohttp = 3.8.4, aiohttp-retry = 2.8.3),
	https (aiohttp = 3.8.4, aiohttp-retry = 2.8.3),
	s3 (s3fs = 2023.6.0, boto3 = 1.26.76)
Config:
	Global: /home/lahorka/.config/dvc
	System: /etc/xdg/xdg-ubuntu/dvc
Cache types: hardlink, symlink
Cache directory: ext4 on /dev/mapper/vgubuntu-root
Caches: local
Remotes: None
Workspace directory: ext4 on /dev/mapper/vgubuntu-root
Repo: dvc, git
Repo.site_cache_dir: /var/tmp/dvc/repo/61d7685c7782209ead53dc8ec2d5f7a5**

```
**Additional Information (if any):**
Running the same workflow as described under _Reproduce_ works when the command used in combination with `--force` is `get` instead of `import`.

**Comment by efiop** [comment]
Hi @LahiLuk , thanks for the feedback! As you've noted, `--force` was never implemented or documented, so this is a feature request really and the error is correct. The complication with `import` is that there might be some existing dvc file that tracks that path already (e.g. from previous import), so you not only need to delete the output but also delete or overwrite corresponding dvcfile (or some hybrid approach). Contributions are welcome 🙂

**Comment by LahiLuk** [comment]
@efiop I would like to try and contribute to implement this feature.

Regarding what you said about the need to overwrite the corresponding `.dvc` file in addition to deleting the data file / folder, I was thinking to take the same approach as with `dvc import-url`, where the `--force` flag has already been implemented.

However, testing in more detail to see how `dvc import-url --force` behaves, I've noticed:
- `--force` flag is only needed if we want to override the local file / folder, the file / folder itself gets deleted and downloaded again, and the .dvc file is overwritten;
- in the case that only a `.dvc` file already exists locally [e.g. created using `dvc import-url --no-download`], it will be overwritten automatically, without a warning or the need to use the `--force` flag.

I was wondering if this was the desired behaviour?
